### ISSUE TITLE
add mention to helpers page in workshop website template

### ISSUE THREAD

**Issue by fmichonneau** [carpentries/instructor-training#1072]
The workshop website template has now a dedicated page (at <url for workshop website>/helpers/index.html, e.g., <https://carpentries.github.io/workshop-template/helpers/index.html>) that has some useful information for helpers. It might be useful to have it mentioned during instructor training.

**Comment by fmichonneau** [comment]
possibly related to #593

**Comment by ChristinaLK** [comment]
Add to "how we operate" or wherever we talk about about running your own workshop.

**Comment by karenword** [comment]
@fmichonneau I need to clarify this before adding to the current updates. While I see that the page you link to exists, it does not appear that websites created using the template have an extras tab that would allow navigation to this content. I also don't see any instructions referring to this content in the workshop website set-up instructions (though text-search may be failing me.) 

How is this page intended to function in the workshop website template? Are they to share a custom link to the hidden page for their helpers, with potentially customized content? If you wouldn't mind proposing some text e.g. that could be included with the website setup activity, or in a discussion of helper roles, that would be very helpful.

**Comment by fmichonneau** [comment]
This page was created at the request of instructors who wanted to share information with their helpers. Instead of sending an email, they could put information in one place and send the link to the helpers. There is no link to this page because it is only intended for the helpers. So instructors (or hosts) add information there and share the links with the helpers.

The link I initially shared is the one from the template but the intent would be for each workshop to customize it with information relevant to their workshops. 

Not having a link to it does make it more difficult for instructors to know it exists.

**Comment by annajiat** [comment]
[Information for Helpers](https://carpentries.github.io/workshop-template/helpers/index.html) page links to four resources:

1. [Helper Checklist](https://docs.carpentries.org/topic_folders/hosts_instructors/hosts_instructors_checklist.html#helper-checklist)
2. [Code of Conduct](https://docs.carpentries.org/topic_folders/policies/code-of-conduct.html)
3. [Key points](https://carpentries.github.io/instructor-training/reference.html) from instructor training
4. [Configuration Problems and Solutions wiki page](https://github.com/swcarpentry/workshop-template/wiki/Configuration-Problems-and-Solutions).


The [Workshop template](https://carpentries.github.io/workshop-template/) has a **Roles** section that links to [Workshop FAQ](https://carpentries.org/workshop_faq/#what-are-the-roles-of-everyone-participating-in-a-workshop) which in terms covers point no 1.

Point 2 and 4 are already on the workshop page.

How about we add point 3 to the Helper Check list?

**Comment by ndporter** [comment]
I agree that it makes sense to remove this (hard to find) page and just make the information more obvious elsewhere, either through @annajiat 's solution or something more directly in the lessons themselves. Changing the master lesson templates is (I presume) a little more involved, but it would just need to be a callout in the instructor notes or something. I tend not to assume that people will use the workshop template for anything more than setup instructions, surveys, and scheduling.
